packages/iTwinUI-react/src/core/Select/Select.test.tsx

```diff
@@ -446,3 +448,88 @@ it('should pass custom props to menu item', () => {
   ) as HTMLElement;
   expect(menuItem.getAttribute('data-value')).toBe('Test one');
 });
+
+it('should select multiple items', () => {
+  const onChange = jest.fn();
+  const { container, rerender } = renderComponent({
+    onChange,
+    multiple: true,
+  });
+
+  const select = container.querySelector('.iui-input-with-icon') as HTMLElement;
+  expect(select).toBeTruthy();
+
+  fireEvent.click(select.querySelector('.iui-select-button') as HTMLElement);
+  const menu = document.querySelector('.iui-menu') as HTMLUListElement;
+  assertMenu(menu);
+
+  let menuItems = menu.querySelectorAll('.iui-menu-item');
+  expect(menuItems.length).toBe(3);
+  fireEvent.click(menuItems[1]);
+  expect(onChange).toHaveBeenCalledWith(1, 'added');
+  fireEvent.click(menuItems[2]);
+  expect(onChange).toHaveBeenCalledWith(2, 'added');
+
+  rerender(
+    <Select<number>
+      options={[...new Array(3)].map((_, index) => ({
+        label: `Test${index}`,
+        value: index,
+      }))}
+      value={[1, 2]}
+      multiple
+      onChange={onChange}
+    />,
+  );
+  menuItems = menu.querySelectorAll('.iui-menu-item');
+  expect(menuItems[1].classList).toContain('iui-active');
+  expect(menuItems[2].classList).toContain('iui-active');
+
+  const tagContainer = select.querySelector('.iui-select-tag-container');
+  expect(tagContainer?.childNodes.length).toBe(2);
+  fireEvent.click(menuItems[2]);
+  expect(onChange).toHaveBeenCalledWith(2, 'removed');
+});
+
+it('should use custom render for selected item (multiple)', async () => {
+  const { container } = render(
+    <Select
+      value={['green', 'red']}
+      selectedItemRenderer={(options) => (
+        <>
+          {options.map((option) => (
+            <span key={option.label} style={{ color: option?.value }}>
+              {option.label}
+            </span>
+          ))}
+        </>
+      )}
+      options={[
+        { value: 'yellow', label: 'Yellow' },
+        { value: 'green', label: 'Green' },
+        { value: 'red', label: 'Red' },
+      ]}
+      multiple
+    />,
+  );
+
+  const selectedValues = container.querySelectorAll(
+    '.iui-select-button > span',
+  );
+  expect(selectedValues.length).toBe(2);
+  expect((selectedValues[0] as HTMLElement).style.color).toEqual('green');
+  expect((selectedValues[1] as HTMLElement).style.color).toEqual('red');
+});
+
+it('should call onChange if tag is removed', () => {
```

Comment:
nit: You tested that in the previous test.

Reply:
no, there I tested clicking on same item, whereas here testing the close button on tag.